src/utils/slack/base.js

```diff
@@ -287,6 +290,28 @@ const getHlxConfigMessagePart = (hlxConfig) => {
   return `, _HLX Version_: *${hlxVersion}*, _Dev URL_: \`https://${rso.ref}--${rso.site}--${rso.owner}.aem.live\``;
 };
 
+/**
+ * Loads profile configuration from JSON file.
+ *
+ * @async
+ * @param {string} profileKey - The profile key to retrieve.
+ * @returns {Object} - The profile configuration object.
+ */
+const loadProfileConfig = (profileKey) => {
+  try {
+    const data = fs.readFileSync(PROFILE_CONFIG_PATH, 'utf-8');
+    const profiles = JSON.parse(data);
+
+    if (!profiles[profileKey]) {
+      throw new Error(`Profile "${profileKey}" not found in ${PROFILE_CONFIG_PATH}`);
```

Comment:
this error seems to be caught locally. should we inform the user (outside this method) if a profile is not found?